pkgs/applications/misc/fff/default.nix

```
@@ -1,24 +1,30 @@
-{ stdenv, fetchFromGitHub, makeWrapper, xdg_utils, file, coreutils }:
+
+{ stdenv, fetchFromGitHub, makeWrapper,
+  bashInteractive, xdg_utils, file, coreutils, w3m, xdotool,
+  imageDisplaySupport ? true
+}:
 
 stdenv.mkDerivation rec {
   pname = "fff";
-  version = "2.0";
+  version = "2.1";
 
   src = fetchFromGitHub {
     owner = "dylanaraps";
     repo = pname;
     rev = version;
-    sha256 = "0pqxqg1gnl3kgqma5vb0wcy4n9xbm0dp7g7dxl60cwcyqvd4vm3i";
+    sha256 = "0s5gi5ghwax5gc886pvbpcmsbmzhxzywajwzjsdxwjyd1v1iynwh";
   };
 
-  pathAdd = stdenv.lib.makeSearchPath "bin" [ xdg_utils file coreutils ];
-  buildInputs = [ makeWrapper ];
+  pathAdd = stdenv.lib.makeSearchPath "bin" ([ xdg_utils file coreutils ] ++ stdenv.lib.optionals imageDisplaySupport [ w3m xdotool ]);
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildInputs = [ bashInteractive ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
 
-  installPhase = ''
-    install -D fff "$out/bin/fff"
-    install -D README.md "$out/share/doc/fff/README.md"
-    install -D fff.1 "$out/share/man/man1/fff.1"
-    wrapProgram $out/bin/fff --prefix PATH : ${pathAdd}
+  postInstall = ''
+    wrapProgram "$out/bin/fff" --prefix PATH : ${pathAdd}
```

```suggestion
    wrapProgram "$out/bin/fff" --prefix PATH : $pathAdd
```
This way if you override `pathAdd` attribute it will be picked by `postInstall`